client/my-sites/earn/memberships/section.tsx

```diff
@@ -83,31 +74,68 @@ function MembershipsSection( { query }: MembershipsSectionProps ) {
 	function renderSettings() {
 		return (
 			<div>
-				<SectionHeader label={ translate( 'Settings' ) } />
-				<Card
-					onClick={ () => setDisconnectedConnectedAccountId( connectedAccountId ) }
-					className="memberships__settings-link"
-				>
-					<div className="memberships__module-plans-content">
-						<div className="memberships__module-plans-icon">
-							<Gridicon size={ 24 } icon="link-break" />
-						</div>
-						<div>
-							<div className="memberships__module-settings-title">
-								{ translate( 'Disconnect Stripe Account' ) }
+				<SectionHeader label={ translate( 'Settings' ) }>
+					{ ! connectedAccountId && (
+						<Button
+							primary
+							compact
+							href={ connectUrl }
+							onClick={ () =>
+								dispatch( recordTracksEvent( 'calypso_memberships_stripe_connect_click' ) )
+							}
+						>
+							{ translate( 'Connect Stripe' ) }
+						</Button>
+					) }
+				</SectionHeader>
```

Comment:
Content of the settings card is now conditional based on if stripe is connected or not.